torch/csrc/jit/fuser/compiler.cpp

```diff
@@ -86,6 +86,9 @@ static const Node* usedInFusedChunk(const Value* input) {
 static void setInputChunkDescriptors(KernelSpec& spec) {
   spec.inputChunks().reserve((spec.graph())->inputs().size());
   for (const Value* input : (spec.graph())->inputs()) {
+    if (!input->type()->isSubtypeOf(DynamicType::get())) {
+      continue;
+    }
```

Comment:
Please document the fact that we only have as many chunk descriptors as tensor inputs. Those things were simple previously, but are getting complicated now.